custom-per-route-options.html.md.erb

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring per-route options
```

Comment:
Questions: 
Is this a new file?
If so, have you raised a PR to add it to the TOC? (docs-book-cloudfoundry)